### PR TITLE
Feat/multiclass node

### PR DESCRIPTION
**Pull request by a-stickan**
# Description

This PR does the following:
- Adds a node for multiclass segmentation
- Adds a launch file for the node

# Testing steps (If relevant)
1. `colcon build && colcon test && colcon test-result --verbose`
2. `. install/setup.bash`
3. `ros2 launch urc_perception multiclass_segmentation.launch.py`

Expectation: Node starts up, but fails due to `No such file or directory: 'Allan/Please/Add/Details'`

# Self Checklist
- [x] I have formatted my code using `ament_uncrustify --reformat`
- [x] I have tested that the new behavior works

### CODE REVIEW

**Review by VAM7686 — CHANGES_REQUESTED** [review]
Nice job! Just have some minor questions.

File: urc_gazebo/launch/simulation.launch.py [hunk 1]
```diff
@@ -29,17 +27,13 @@ def generate_launch_description():
         )
     )
 
-    parameters_file_path = os.path.join(get_package_share_directory(
-        'urc_gazebo'), 'config', 'urc_gazebo_params.yaml')
-    with open(parameters_file_path, 'r') as file:
-        scan_to_pointcloud_params = yaml.safe_load(file)['scan_to_pointcloud']['ros_parameters']
-
     scan_to_pointcloud = Node(
```
[the remark this answers is not in the document]
Comment (a-stickan) [reply urc_gazebo/launch/simulation.launch.py:30]: I don't think we do, removing before I merge this branch in

File: urc_navigation/config/ekf_localization_node_params.yaml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    # numeric_limits<double>::max() if unspecified. It is strongly recommended that these parameters be removed if not
+    # required. Data is specified at the level of pose and twist variables, rather than for each variable in isolation.
+    # For messages that have both pose and twist data, the parameter specifies to which part of the message we are applying
+    # the thresholds.
+    #odom0_pose_rejection_threshold: 5.0
+    #odom0_twist_rejection_threshold: 1.0
+
+    # Further input parameter examples
+    odom1: /odometry/gps
+    odom1_config: [ true, true, false,
```
[the remark this answers is not in the document]
Comment (a-stickan) [reply urc_navigation/config/ekf_localization_node_params.yaml:130]: We will be performing 3D localization, must have just set the param to true by accident. Fixed in most recent commit

**Review by VAM7686 — APPROVED** [review]
looks good to me!

**Pull request merged**